configure.ac

```diff
@@ -267,29 +255,96 @@ dnl --- enable_pkey
 enable_pkey=no
 
 case $target in
-     *s390x*)
-	  enable_pkey=yes
-	  ;;
-     *s390*)
-	  CFLAGS="$CFLAGS -m31 -DNO_PKEY"
-	  ;;
-     *ppc64* | *x86_64*)
-	  CFLAGS="$CFLAGS -DNO_PKEY"
-	  ;;
-     *ppc* | i*86*)
-	  CFLAGS="$CFLAGS -m32 -DNO_PKEY"
-	  ;;
-     *)
-	  CFLAGS="$CFLAGS -DNO_PKEY"
-	  ;;
+	*s390x*)
+		enable_pkey=yes
+	;;
+	*s390*)
+		CFLAGS="$CFLAGS -m31"
+		AC_DEFINE([NO_PKEY])
+	;;
+	*ppc64* | *x86_64*)
+		AC_DEFINE([NO_PKEY])
+	;;
+	*ppc* | i*86*)
+	  CFLAGS="$CFLAGS -m32"
+		AC_DEFINE([NO_PKEY])
+	;;
+	*)
+		AC_DEFINE([NO_PKEY])
+	;;
 esac
 AM_CONDITIONAL([NO_PKEY], [test "x$enable_pkey" != "xyes"])
 
+dnl Check and save whether the platform is AIX
+dnl Using the pthread library requires the _THREAD_SAFE macro to be set
+build_aix=no
+case "${host_os}" in
+	aix*)
+		build_aix=yes
+		AC_DEFINE([_ALL_SOURCE])
+		AC_DEFINE([_THREAD_SAFE])
+		with_systemd=no
+		with_libica=no
+		enable_ccatok=yes
+		enable_swtok=yes
+		enable_tpmtok=no
+		enable_icsftok=no
+		enable_ep11tok=no
+		echo "Disabling migration utilities on AIX: pkcscca pkcstok_migrate"
+		enable_pkcscca=no
+		enable_pkcstok_migrate=no
+		echo "Disabling the following unsupported features on AIX: systemd libica pmtok icsftok ep11tok"
```

Comment:
did you intentionally remove the 't' from 'tpmtok' here?

Reply:
Nope, that is accidental! Fixing that.